docs/120-help-desk/working-practices.md

```diff
@@ -34,29 +38,26 @@ Often when someone asks a question about a web application, responding quickly m
 
 The first step toward helping users feel attended-to is to **always reply promptly**. Any reply is better than no reply. Users and clients like to know that we've read their correspondence, and that we're looking into the issue. We don't have to know everything in order to begin an interaction, and we don't even have to be specific about next steps; often our first response may be as simple as:
 
-```
-Hi (name),
-Thanks for sending this over! It sounds like you're experiencing a problem with (statement of problem). We'll (research some more about this / put it in front of our engineers immediately / see if I can solve that problem), and we'll follow up once we've got more information.
-
-Let me know if I can help with anything else in the meantime!
+_Hi (name),
+Thanks for sending this over! It sounds like you're experiencing a problem with (statement of problem). We'll (research some more about this / put it in front of our engineers immediately / see if I can solve that problem), and we'll follow up once we've got more information._
 
+_Let me know if I can help with anything else in the meantime!_
 
-Thanks,
+_Thanks,
 (My name)
-CivicActions Support
-```
+CivicActions Support_
```

Comment:
@VanessaLuxen looks like our automated tool thinks this is a heading. Do you want try putting the example response in a blockquote?

Reply:
A preview of you changes can be seen here https://handbook.civicactions.com/en/vanessaluxen-patch-3/120-help-desk/working-practices/